config/dcp.go

```diff
@@ -41,11 +47,12 @@ type DCPListener struct {
 }
 
 type ExternalDcp struct {
-	Group                DCPGroup      `yaml:"group"`
-	BufferSize           int           `yaml:"bufferSize"`
-	ConnectionBufferSize uint          `yaml:"connectionBufferSize"`
-	ConnectionTimeout    time.Duration `yaml:"connectionTimeout"`
-	Listener             DCPListener   `yaml:"listener"`
+	ConnectionBufferSizeMB string        `yaml:"connectionBufferSizeMB"`
```

Comment:
I don't think we need to mention MB here since we can mention it in readme. what do you think about it bro?